articles/postgresql/concepts-connection-libraries.md

```diff
@@ -11,10 +11,10 @@ ms.topic: article
 ms.date: 09/26/2017
 ---
 # Connection libraries for Azure Database for PostgreSQL
-This topic lists libraries and drivers that developers can use to program applications to connect to and query Azure Database for PostgreSQL.
+This topic lists libraries and drivers that developers can use to develop applications to connect to and interact with Azure Database for PostgreSQL.
```

Comment:
The libraries and drivers can be used for more than just queries, so changed the text to "interact with"

Reply:
Its right, but who searches the internet for how to interact with a database? Thinking SEO may lend some points to query.